http3/server.go

```diff
@@ -267,7 +267,7 @@ func (s *Server) Serve(conn net.PacketConn) error {
 
 // ServeQUICConn serves a single QUIC connection.
 func (s *Server) ServeQUICConn(conn quic.Connection) error {
-	return s.handleConn(conn)
+	return s.handleConn(context.Background(), conn)
```

Comment:
I don't think this is necessary. It will cause the quic connection to be shutdown. Users should coordinate this with the server shutdown, or not call shutdown at all.